transformer_lens/HookedTransformer.py

```diff
@@ -1442,18 +1445,22 @@ def load_and_process_state_dict(
                 )
             else:
                 state_dict = self.center_writing_weights(state_dict)
+
         if center_unembed:
             state_dict = self.center_unembed(state_dict)
         if fold_value_biases:
             state_dict = self.fold_value_biases(state_dict)
         if refactor_factored_attn_matrices:
             state_dict = self.refactor_factored_attn_matrices(state_dict)
-        self.load_state_dict(state_dict)
+
+        self.load_state_dict(state_dict, strict=False)
 
     def fill_missing_keys(self, state_dict):
         return loading.fill_missing_keys(self, state_dict)
 
-    def fold_layer_norm(self, state_dict: Dict[str, torch.Tensor]):
+    def fold_layer_norm(
+        self, state_dict: Dict[str, torch.Tensor], fold_biases=True, center_weights=True
+    ):
         """Fold Layer Norm.
 
         Takes in a state dict from a pretrained model, formatted to be consistent with
```

Comment:
Can you add a comment saying that this is both used for LayerNorm and RMSNorm?

Reply:
added